enterprise/cmd/frontend/internal/authz/resolvers/resolver.go

```diff
@@ -198,10 +198,14 @@ func (r *Resolver) ScheduleUserPermissionsSync(ctx context.Context, args *graphq
 		return nil, err
 	}
 
-	err = r.repoupdaterClient.SchedulePermsSync(ctx, protocol.PermsSyncRequest{
+	req := protocol.PermsSyncRequest{
 		UserIDs: []int32{userID},
-	})
-	if err != nil {
+	}
+	if args.Options != nil && args.Options.InvalidateCaches != nil {
```

Comment:
`args.Options.InvalidateCaches` can still be `false` while `args.Options.InvalidateCaches != nil`, that is a legitimate value.
```suggestion
	if args.Options != nil && args.Options.InvalidateCaches != nil && args.Options.InvalidateCaches {
```

Reply:
I swear I got caught by this before as well haha, thanks!